src/index.html

```diff
@@ -8,9 +8,53 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+      action="/"
+      method="get"
+      data-qa="big"
```

Comment:
Please fix this in the whole file.
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/54860641/e5629df2-59c6-49cf-a116-bf9ce0a8cb59)